### DISCUSSION TITLE
Please add  wiki

### DISCUSSION THREAD

**Discussion by hideyasu2015** [community/community#130299] [Mobile]
I want to send wiki url from my iPhone

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @hideyasu2015 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your feedback entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
